scripts/protoc-gen-depends

```diff
@@ -81,4 +88,4 @@ def add_depends(fd):
 
 res.file.extend([add_depends(fileDescriptor) for fileDescriptor in req.proto_file])
 
-stdout.write(res.SerializeToString())
```

Comment:
Yes, this also gives me errors. The problem is that the bytes that are in stdin aren't necessarily text, it's a stream of bytes. In my case it was byte `0xc6` that couldn't be decoded. The error occurs on `stdin.readlines()`, because there it'll interpret stdin as text: it'll try to decode the bytes from stdin as utf-8. It shouldn't at all decode the bytes: it should keep the bytes as-is as stdin isn't necessarily compatible with utf-8 (or whatever encoding is used).

stdin.buffer is a stream of bytes, instead of stream of text. That's why that does work (it doesn't decode the bytes at all). There just isn't a handy function (that I've found) to just read all bytes from stdin.buffer as one big bytestring, which makes it seem a lot more complicated than it should be.